### PR TITLE
[Doc Improvement][ME Copilot plugin for m365]

### PR DESCRIPTION
**Pull request by v-shalinir**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/me-copilot-plugin-for-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/534a2808ffbb46e1e420c461d4447badf491e8c9/msteams-platform%2Fmessaging-extensions%2Fme-copilot-plugin-for-m365.md) | 76 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c34134dc-bb89-4a6f-8a32-13882859d158) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/me-copilot-plugin-for-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1a232b436fcb4c5ed655248e188f48f390c2bf56/msteams-platform%2Fmessaging-extensions%2Fme-copilot-plugin-for-m365.md) | 82 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6edc0321-3b18-4b5e-b1a8-dad0cf158cd4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/me-copilot-plugin-for-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/21a59b164d69f31044a4249c93a2d00ed76e3b8b/msteams-platform%2Fmessaging-extensions%2Fme-copilot-plugin-for-m365.md) | 82 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/893e3e17-a566-4b2a-b304-e7f8340f0c59) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/me-copilot-plugin-for-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f49c0d5d65abf32abd8301e0b5defe1262ead9cd/msteams-platform%2Fmessaging-extensions%2Fme-copilot-plugin-for-m365.md) | 82 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/53c992fe-8542-40b2-8bea-c82f0b85b185) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0a54380b9b136ba8fc7a355c19dbae2ac2349f63/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 91 | 45 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/437ce03b-46ff-423c-8fa7-57dca11f1a1a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/me-copilot-plugin-for-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0a54380b9b136ba8fc7a355c19dbae2ac2349f63/msteams-platform%2Fmessaging-extensions%2Fme-copilot-plugin-for-m365.md) | 82 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6014b7bc-eb9e-4d9d-af45-3b8022ac32c1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c60f9340461ec9f527b21495ae892027f960ef8/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 91 | 45 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d40d8430-daf4-46d0-803b-57a665db2121) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/me-copilot-plugin-for-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c60f9340461ec9f527b21495ae892027f960ef8/msteams-platform%2Fmessaging-extensions%2Fme-copilot-plugin-for-m365.md) | 82 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1177a88c-ea93-4983-99d6-cf341bcf8397) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3e840c253310c434fb1d79351bd4d571bdf5f744/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 91 | 45 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/68ddf07d-e487-4f6a-97c7-8ad8af7a6121) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4499aa3915f33cbd4bbfde1f8974a983b547bf9e/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 91 | 43 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d58afbf4-e937-4f7c-9530-cfcb6cfedfba) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f10582c1d243a40b6164970e361ebdfeb3b7c46a/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 91 | 43 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/66e64235-8584-4fa8-b048-df9a5cef405d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/222a1c03a22cc724de774e05ec52727468652fe0/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3442ab5c-eccc-4161-92f4-5a8d96f0b800) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a2c1c2c87f42e3162bb3d0713e4ef29b7d0e8835/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2fbb8695-5ac8-486c-9c2a-27737fb3e64a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/high-quality-message-extension.md [hunk 1]
```diff
@@ -751,16 +751,77 @@ For a plugin to be validated, invoked, and work seamlessly, ensure that it meets
...
+
+    1. Sign in to [Azure portal](https://portal.azure.com/)  with your sandbox tenant account. 
+    1. Open <b>App registrations</b>. 
+    1. Select the name of your application to open its app registration. 
+    1. Select Expose an <b>API</b> (under Manage). 
+    1. In the <b>Authorized client applications</b> section, ensure the following Client Id values are listed:
+
+        | Microsoft 365 client application  | Client ID |
+        | --- | --- |
+        | OfficeAIAppChatCopilotExt | 3068386c-7a16-4f6a-a664-043b6b232816 |
```
Comment (deepeshsatija1012) [inline msteams-platform/messaging-extensions/high-quality-message-extension.md:787]: Can we rename `OfficeAIAppChatCopilotExt` to `Word, PowerPoint, Excel (Web, Desktop)`

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7881cf2d7e31e59a50d89ce9f5f2dae1a2cf03cb/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/221e40ea-c144-4828-afd5-5f9c31ee861b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6eecb9af8283e79503552c3fc295b9ce80e9a279/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3a4ab143-01e6-4c41-a3c3-5644b7854755) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0d9fb1a87bd1b10d7c34908a36d99e19b8c2bede/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 41 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d6101d8f-77f2-4914-b973-ab82026f8d4a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d61b05d9ffa9f05b590bb8875f75a561bcb36563/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 41 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/89b93892-a7b6-4096-92f7-430d5a97ba55) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d74dadd9aad5c98e3ed40221b948cf31dcae464b/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 41 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9f3809de-d4e2-47d0-83c9-db1bab8f9421) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e1d6d957ad3b19ea293dc2032676a9693a989cf9/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 43 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1c1e1006-1afa-44ec-87f6-92b994177def) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3927ea64f5eb793b90c1c41cbc30cfb329fd7b71/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 43 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e5119147-183b-4805-b0d7-cffb78ea9e8d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0997b7742887771558d41d012d301ae072629c4e/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 43 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0f12124b-aceb-4915-b2c4-3ec2cd3fc2c7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/high-quality-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d1435a9a417ef66d004b7c5fc5676050199a8820/msteams-platform%2Fmessaging-extensions%2Fhigh-quality-message-extension.md) | 92 | 43 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2c9e0ffb-3786-4941-a3e1-b8b5c334f8bd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**